templates/typescript-mcp/AGENTS.md

```
@@ -0,0 +1,83 @@
+`moose generate hash-token` outputs a key pair — the hash goes to the backend and the token goes to the frontend:
+
+| Variable | File | Value |
+| --- | --- | --- |
+| `MCP_API_KEY` | `packages/moosestack-service/.env.local` | `ENV API Key` (hash) |
+| `MCP_API_TOKEN` | `packages/web-app/.env.local` | `Bearer Token` |
+| `ANTHROPIC_API_KEY` | `packages/web-app/.env.local` | Anthropic API key |
+| `MCP_SERVER_URL` | `packages/web-app/.env.development` | Pre-set to `http://localhost:4000` |
+
+## Custom MCP Tools (at `/tools`)
+
+### `query_clickhouse`
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)